ipfscluster_test.go

```diff
@@ -59,6 +60,7 @@ var (
 
 	consensus = "crdt"
 	datastore = "badger"
+	connector = "arbiter"
```

Comment:
@hsanjuan I will change this back to the normal ipfshttp connector, once you are happy with the PR.